kiwi/defaults.py

```diff
@@ -602,7 +602,8 @@ def get_signed_grub_loader(root_path):
         signed_grub_file_patterns = [
             '/usr/share/efi/*/grub.efi',
             '/usr/lib64/efi/grub.efi',
-            '/boot/efi/EFI/*/grub*.efi'
+            '/boot/efi/EFI/*/grub*.efi',
+            '/usr/share/grub*/arm64-efi/grub.efi'
```

Comment:
This should probably be `/usr/share/grub*/*-efi/grub*.efi` There's probably going to be other architectures that will get this treatment...

Reply:
yes fully agree, done now. The integration test for secure boot on suse aarch64 has completed here:

* https://build.opensuse.org/package/show/Virtualization:Appliances:Images:Testing_arm/test-image-secure-boot-iso

Now we just have to make Fedora build. Just in case I couldn't get help from obs, do you have some aarch64 machine running with Fedora somewhere ? If not I'll check if I can run an a1 instance in AWS with Fedora. Not sure if there is an offering available